app-modules/portal/src/Http/Middleware/AuthenticateIfRequiredByPortalDefinition.php

```diff
@@ -51,8 +54,14 @@ public function handle(Request $request, Closure $next): Response
             return $next($request);
         }
 
-        if (! auth('sanctum')->check()) {
-            abort(403);
+        $user = app(Guard::class)($request);
+
+        if (! $user instanceof Student && ! $user instanceof Prospect) {
```

Comment:
Shouldn't this be an `||`? Would this not fail since the `$user` would only ever been an instance of a `Student` or a `Prospect` not both?

Could we just check if is an instance of `Educatable`?